Source/Scene/Cesium3DTileset.js

```diff
@@ -1237,6 +1239,23 @@ define([
             get : function() {
                 return this._ellipsoid;
             }
+        },
+
+        /**
+         * Returns the `extras` property at the top-level of the tileset JSON, which contains application specific metadata.
```

Comment:
Instead of tick marks use `<code>extras</code>` in documentation for code highlighting.